apache/superset
# docs: add GA conf to config

### SUMMARY

A recent migration of the docs site to Docusaurus in #17411 lost Google Analytics tracking. Here we add the same GA confs that were introduced in #11613 by leveraging the [`plugin-google-analytics`](https://docusaurus.io/docs/using-plugins/#docusaurusplugin-google-analytics) plugin that's included in the [`preset-classic`](https://docusaurus.io/docs/using-plugins/#docusauruspreset-classic) that's already in use. In addition, Docusaurus is bumped to the latest version as the configuration mechanism changed recently (see https://github.com/facebook/docusaurus/pull/5832 which was included in the `2.0.0-beta.10` release). A missing license property is added to the `package.json` file of the docs package.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
The site looks the same as before:
<img width="1791" alt="image" src="https://user-images.githubusercontent.com/33317356/155276541-2cb45fb9-62ca-4397-bd97-e0a9157a4404.png">

### TESTING INSTRUCTIONS
Make sure the docs site builds correctly and looks correct when run locally:

1. `cd docs`
2. `yarn install`
3. `yarn build`
4. `yarn start`



### ADDITIONAL INFORMATION
